Gemfile

```diff
@@ -84,6 +86,7 @@ group :development, :test do
   gem 'guard-rspec', require: false
 
   gem 'factory_bot_rails'
+  gem 'factory_bot', '4.10.0'
```

Comment:
`factory_bot_rails` always includes the same specified version of `factory_bot`, so you can just remove the `factory_bot` listing and just specify the version on `factory_bot_rails`.  i.e.
> gem 'factory_bot_rails', '4.10.0'

Reply:
changed